packages/ove-service-layout/docs/WORKING_EXAMPLE.md

```diff
@@ -0,0 +1,105 @@
+# Working example
+
+```json
+{
+  "hasVideos": false,
+  "ove-space": "http://localhost:3004/space-info",
+  "canvas": {
+    "layout": {
+      "type": "static"
+    },
+    "sections": [
+      {
+        "name": "container1",
```

Comment:
@oserban, section names should be optional. Is this always the case, and is it tested appropriately?

Reply:
If names are optional, validation and error reporting is going to be a nightmare. We can auto-generate them from the UI and sdks, so I don't see this as an issue.

Reply:
ok, I recalled the conversation we had on this. Sometimes we can only have one out of two options and I am happy to stick to the `name must be provided` convention. 

Anyway this would mean that we have to fix all of the existing defintions of sections to have some arbitrary value for as their name. Given that this is a requirement driven by error reporting, isn't it possible to say print out the entire section configuration, if the name was not specified?

Reply:
@oserban, when you said it is going to be a nightmare from the validation point of view: if it cannot support an optional parameter, then lets stick to what you have. If it does, and this is simply because you couldn't figure out a way of reporting a validation error etc - then, please re-consider this.

Reply:
@senakafdo I ca do the reporting, but it will not make much sense in certain situations. I can make a script to generate names for all the current configuration so we can easily migrate all the apps.